packages/dashboard/README.md

```diff
@@ -1,3 +1,3 @@
 # dashboard
 
-This component is a wrapper for intended for use with [fabric dashboard cards](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/dashboard/src/components/Card). Please follow the example to see usage and guidelines.
\ No newline at end of file
+We've moved the Dashboard code to a new dedicated repo and have deprecated this experimental package. Please contact m365admincomponents@microsoft.com for information about using and contributing to these controls.
```

Comment:
perfect....This is great. The alias email definitely feels good here.

Reply:
Also just to note that we still need to deprecate the package within NPM manually, right? 

https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

Reply:
Yes we do. I'll get David's help to do that later.